x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -88,21 +85,21 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
 
       it('finds data after load and polling', async () => {
-        await esArchiver.load('endpoint/metadata/destination_index', { useCreate: true });
-        await pageObjects.endpoint.waitForTableToHaveData('endpointListTable', 1100);
+        await esArchiver.load('endpoint/metadata/api_feature', { useCreate: true });
```

Comment:
these tests are skipped right now, so it's OK for now, but I'd rather we not wait for the transform in the integration tests because of the long wait times.

IMO, the UI integration tests can just test that we can call the AP when there's data there and the UI renders correctly

Reply:
Well that should be tested in unit tests, the integration test should be integration. And ingesting into metadata current may not be a good test but I get the point. To get the test to pass again I had to change a few line that were removed.

Reply:
Also the old setup will fail since we remove the matadata current stream.